data/projects.js

```diff
@@ -147,5 +147,127 @@ coala should support generating metrics for your code.",
 		"initiatives" : ["GSoC"],
 		"tags" : ["Web", "Blog"],
 		"markdown": "coala_web.md"
+	},
+	{
+		"name" : "Optimize Caching",
+		"desc" : "Improve coala's performance by implementing caching strategies.",
+		"requirements" : [
+			"At least one patch to the coala core should be accepted and merged.",
+			"The student should be familiar with ongoing technical cEPs, \
+			especially \
+			[cEP-0002](https://github.com/coala/cEPs/blob/master/cEP-0002.md).",
+			"The proposal needs to show some familiarity with coala's caching \
+			implementation.",
+		],
+		"difficulty" : "medium",
+		"issues" : [
+			"https://github.com/coala/coala/issues/2541",
+			"https://github.com/coala/coala/issues/2699",
+			"https://github.com/coala/coala/issues/2912"
+		],
+		"mentors" : [
+			"adtac",
+			"sils"
+		],
+		"initiatives" : ["GSoC"],
+		"tags" : ["Core", "Bears"],
+		"markdown": "optimize_caching.md"
+	},
+	{
+		"name" : "Cohesive Editor Support",
+		"desc" : "Develop an efficient strategy that allows coala to maintain \
+		a number of plugins for different editors.",
+		"requirements" : [
+			"The proposal has to provide a convincing strategy for testin, \
+			 documentation and maintaining editor plugins",
+			"The applicant should have one patch accepted to any of the coala \
+			plugins."
+		],
+		"difficulty" : "medium",
+		"issues" : [
+			"https://github.com/coala/coala-atom/issues/37",
+			"https://github.com/coala/coala-emacs/issues/11",
+			"https://github.com/coala/coala-eclipse/issues/25",
+			"https://github.com/coala/coala-vim/issues/24",
+			"https://github.com/coala/coala-gedit/issues/5",
+			"https://github.com/coala/coala-sublime/issues/4"
+		],
+		"mentors" : [
+			"AbdealiJK",
+			"fneu"
+		],
+		"initiatives" : ["GSoC"],
+		"tags" : ["Editors", "Plugins"],
+		"markdown": "editor_support.md"
+	},
+	{
+		"name" : "Python dead code fixer",
```

Comment:
CC @jendrikseipp creator of vulture: TLDR we're applying for GSoC and we want to get the ability of removing dead code automatically (it'll be proposed to the user so he can approve/reject the patch). I wonder if we could have a GSoC student working on vulture with @jendrikseipp as (co?) mentor instead.